python/doc/examples/reliability_sensitivity/estimate_probability_randomized_qmc.ipynb

```diff
@@ -11,12 +11,22 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this basic example we are going to estimate a failure probability.\n",
+    "In this example we are going to estimate a failure probability."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
     "\n",
     "Let's consider the following analytical example of a cantilever beam, of Young's modulus E, length L, section modulus I.\n",
     "\n",
     "One end is built in a wall and we apply a concentrated bending load F at the other end of the beam, resulting in a deviation:\n",
-    "$$d = \\frac{F*L^3}{3*E*I}$$\n",
+    "\n",
+    "$$\n",
+    "d = \\frac{FL^3}{3EI}\n",
+    "$$\n",
     "\n",
     "It is considered that failure occurs when the beam deviation is too important:\n",
     "\n",
```

Comment:
In the next few lines, I believe "the" should be systematically removed.

Reply:
Fixed.